pkg/morph/subscriber/subscriber.go

```diff
@@ -143,6 +155,14 @@ func (s *subscriber) routeNotifications(ctx context.Context) {
 				}
 
 				s.blockChan <- b
+			case response.NotaryRequestEventID:
+				notaryRequest, ok := notification.Value.(*response.NotaryRequestEvent)
```

Comment:
I see that removed requests are being filtered later, but anyway, do we need them at all?

Reply:
To be fair that's the question I'm asking myself in general (subscription API-wise), who needs them?

Reply:
just wanted to make base functionality in node listener: it can listen to, parse and handle any notary requests

but yes, there are no parsers and handlers for such events in the alphabet node now

/cc @alexvanin

Reply:
I think it still can be a nice indicator of the case when neither main nor fallback transaction was completed. Other cases are covered by existing subscriptions API, e.g. subscriptions for transactions execution (main or fallback).